apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -207,41 +252,57 @@ var TopInstructions = React.createClass({
                 headerHasFocus={false}
                 onClick={this.handleDocumentationClick}
               />}
+            {experiments.isEnabled('resourcesTab') &&
+              <div style={styles.helpTab}>
+                <a className="instructionsTab" onClick={this.handleInstructionTabClick} style={styles.highlighted}>{msg.instructions()}</a>
+                <a className="helpTab" onClick={this.handleHelpTabClick} style={styles.tab}>{msg.helpTips()}</a>
```

Comment:
Nit: For elements with multiple attributes, we prefer one attribute per line (especially when they're getting this long). We also prefer to avoid using CSS classes as much as possible inside our JSX, which should be possible once you've removed your jquery :)